kb/temporal-platform-limits-sheet.md

```diff
@@ -43,3 +43,15 @@ Hard limits fail with an error; soft limits produce a warning log on the server
     - `limit.numPendingCancelRequests.error`
     - `limit.numPendingChildExecutions.error`
 - [Search Attributes maximums](/visibility/#search-attributes-maximums)
+
+**Default limits for Temporal Cloud**
+
+- **Account level**
+  - Namespaces: 10
+  - Prometheus endpoint Retention Period: 1 month
+- **Namespace level**
+  - Actions per second: 200
+  - Certificates: 32-KB payload or 16 certificates, whichever is smaller
+  - Concurrent Task pollers: 2,000 (configurable to a maximum of 100,000)
+  - Custom Search Attributes: 20; maximum of 5 for text
+  - Retention Period: 7 days (configurable to a maximum of 90 days)
```

Comment:
IMO we should state that Retention Period: 7 days ( configurable between 1-90 days )